deploy/crds/openebs_v1alpha1_device_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: openebs.io/v1alpha1
+kind: Device
+metadata:
+  name: example-device
+spec:
+  # Add fields here
+  size: 3
```

Comment:
Can we use the capacity (instead of size?) notation as used by PVC -- and Disk CR in earlier releases.

Reply:
"device_types.go" has capacity. This is stale cr. I am in process of updating sample YAML for disk, device and devicerequest.